src/parser/core/modules/Gauge/CounterGauge.ts

```diff
@@ -10,6 +11,8 @@ interface CounterHistory {
 	maximum: number
 }
 
+export type CounterGaugeModifier = Partial<Record<Event['type'], number>>
```

Comment:
Not super keen on this. It's not actually a counter gauge modifier - that'd just be a number. It doesn't actually represent any type in this file, but rather, the implementation details of a few consumers.

Given it's as trivial as it is, it should probably be inlined into the files that use it - you might be able to narrow the key type, too, which would potentially let you drop the partial